build.gradle

```diff
@@ -51,10 +51,10 @@ dependencies {
     compile project(':commcare')
     compile fileTree(dir: 'app/libs', include: '*.jar')
     compile(name: 'htmlspanner-custom', ext: 'aar')
-    compile 'com.android.support:cardview-v7:23.2.1'
-    compile 'com.android.support:recyclerview-v7:23.2.1'
-    compile 'com.android.support:support-v4:23.2.1'
-    compile 'com.android.support:gridlayout-v7:23.2.1'
+    compile 'com.android.support:cardview-v7:23.3.0'
```

Comment:
@phillipm Since these have been finicky in the past, do you think we need to test that everything looks fine with the new versions?